src/core/tsi/ssl_transport_security.h

```diff
@@ -35,6 +35,19 @@
 
 #define TSI_SSL_ALPN_SELECTED_PROTOCOL "ssl_alpn_selected_protocol"
 
+/* --- ssl_root_certs_store object ---
+
+   This object is a wapper to X509_STORE that stores SSL root certificates.
```

Comment:
X509_STORE: No openSSL mentions in the header please. That breaks the abstraction.